### ISSUE TITLE
DescriptionList - lists should have unique aria-labels

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11392]
**Describe the enhancement or change**
In pages where multiple DescriptionLists are rendered, each should have a unique aria-label to follow the recommended approach for AT that show each list on a page i.e. in VoiceOver's rotor menu. When there's more than 1 on a page, you see a list of just "definition list" which makes it difficult to traverse via that menu.

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

**Comment by Mash707** [comment]
@thatblindgeye could you elaborate on this more. Are we talking about adding unique aria-labels to the examples on pages like this https://www.patternfly.org/components/description-list/ ?

**Comment by Mash707** [comment]
@thatblindgeye could you assign this to me, I'll work on this. Thank you.
